src/Test.UnitTests.Sarif.Multitool.Library/MergeCommandTests.cs

```diff
@@ -48,11 +48,18 @@ public void MergeCommand_WhenThereAreDuplicatedResults_ProducesNonDuplicatedResu
             RunTest("DuplicatedResults.sarif");
         }
 
-        protected override string ConstructTestOutputFromInputResource(string inputResourceName, object parameter)
+        [Fact]
+        public void MergeCommand_WhenPassNoFolderOnlyFile_ProducesCorrectResults()
         {
-            const string InputFolderPath = @"C:\input";
```

Comment:
>@"C:\input" [](http://example.com/codeflow?start=43&length=11)

There is a lot of them in the whole solution,
created a issue to find all windows only style code and fix if possible:
[https://github.com/microsoft/sarif-sdk/issues/2421](https://github.com/microsoft/sarif-sdk/issues/2421)